### DISCUSSION TITLE
Vimeo player controls vanish permanently soon after they first hide

### DISCUSSION THREAD

**Discussion by zeusstl** [vimeo/player.js#979] [Bug reports]
### Expected Behavior

Vimeo player should always show the player controls (including whichever controls the Vimeo account has active: play/pause, fullscreen, bar, etc) whenever the user taps on the video.

### Actual Behavior

1) The player loads.
2) The controls get automatically hidden after a few seconds as they typically would.
3) Tapping on the video does not revive the controls.
4) The fullscreen button never shows even though the width is bigger than 375px (so it should not be in mini or micro modes).

### Steps to Reproduce

Load the player in a webkit browser on Android. This issue does not exist on iOS - only on Android.

Also, tried it with iframe instead of the div with the same results.

Using:

import Player from '@vimeo/player';
...
    const options = {
        id: this.videoId,
        width: window.innerWidth * 0.96,
    };
    const player = new Player( this.el.nativeElement,  options );
...
`<div data-vimeo-autoplay="false" data-vimeo-playsinline="false" data-video-id="#########" data-vimeo-initialized="true"></div>`
Where ######### was an actual video id.

==============
WHEN FIRST CLICKING PLAY
==============

[Deprecation] 'HTMLVideoElement.webkitSupportsFullscreen' is deprecated. Please use 'Document.fullscreenEnabled' instead.
videoEnabled @ player.js:2
[Deprecation] 'HTMLVideoElement.webkitEnterFullscreen()' is deprecated. Please use 'Element.requestFullscreen()' instead.
i @ player.js:2
[Deprecation] 'HTMLVideoElement.webkitDisplayingFullscreen' is deprecated. Please use 'Document.fullscreenElement' instead.
get @ player.js:2
C @ player.js:2
R @ player.js:2
sentryWrapped @ sentry.min.js:2

==============
WHEN FIRST LOADING
==============

Additionally, there are these errors when the player first loads, although it doesn't prevent the player or the controls (besides fullscreen) from loading, so I suspect is unrelated:

- The deviceorientation events are blocked by feature policy. See https://github.com/WICG/feature-policy/blob/master/features.md#sensor-features

- Refused to load the image 'android-webview-video-poster:default_video_poster/3945440453042126112' because it violates the following Content Security Policy directive: "img-src 'self' data: https://i.vimeocdn.com https://secure-b.vimeocdn.com https://f.vimeocdn.com https://vimeo.com https://secure.gravatar.com https://i0.wp.com https://i1.wp.com https://i2.wp.com https://pagead2.googlesyndication.com https://player.vimeo.com https://*.ci.vimeows.com https://f.vimeocdn.com".

Related to: https://github.com/vimeo/player.js/issues/98

**Comment by zeusstl** [comment]
Does anyone have any thoughts on why this would happen?

**Comment by zeusstl** [comment]
Hello, Is there anyone here who can help with this concern?

**Comment by luwes** [comment]
Thanks for your feedback @zeusstl, we will look into this issue.

**Comment by luwes** [comment]
@zeusstl is this still an issue on your end? our QA team couldn't reproduce

**Comment by PhilippeKuret** [comment]
Anyone has a solution on this one?

**Comment by PhilippeKuret** [comment]
I have the same problem and I've been racking my brain on fixing it

**Comment by reelhero** [comment]
same issue - vimeo player controls instantly vanish and dont come back - anyone?

**Comment by gomdolkim** [comment]
same issue

**Comment by luwes** [comment]
please provide some more information as we can't reproduce this issue at the moment.
do you have webpage online where this happens?

**Comment by hourdays** [comment]
same issue when navigating using a mobile (iphone) to the vanilla html and javascript page where I use the player sdk

**Comment by hourdays** [comment]
To be double checked but it seems that adding `<meta name="viewport" content="initial-scale=1">` in the html head fixes the issue

**Comment by victorkifer** [comment]
same issue (Android)

**Comment by bdougherty** [comment]
If anyone is still facing this issue, please provide some more information and an example so that we can try to reproduce.
